### PR TITLE
UPDATE: arrestincoupling1.0

### PR DESCRIPTION
**Pull request by esguerra**

### CODE REVIEW

File: signprot/urls.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
     path('', RedirectView.as_view(url='gprotein', permanent=False), name='index'),
     path('gprotein', views.BrowseSelection.as_view(), name='index'),
     path('arrestin', views.ArrestinSelection.as_view(), name='arrestin'),
-    path('arrestincouplings', cache_page(60*60*24*7)(ArrestinCoupling.as_view()), name='arrestin_coupling'),
+    path('arrestincouplings', ArrestinCoupling.as_view(), name='arrestin_coupling'),
```
Comment (AJK-dev) [inline signprot/urls.py:12]: There is no caching right now. Could you enable it again?

File: build/management/commands/build_arrestins.py [hunk 2]
```diff
@@ -10,6 +10,7 @@
 
 from django.conf import settings
 from django.core.management.base import BaseCommand
+from django.core.management.color import no_style
```
Comment (AJK-dev) [inline build/management/commands/build_arrestins.py:13]: If I'm not mistaken this commit was also part of the other PR right?

**Review by AJK-dev — CHANGES_REQUESTED** [review]
Looking good, only the caching should be enabled again. Will the +/- GRK be part of the next PR?

File: build/management/commands/build_arrestins.py (hunk 2 above)
Comment (esguerra) [reply build/management/commands/build_arrestins.py]: yes. now merged ("rebased").

**Comment by esguerra** [comment]
AJK-dev>> Will the +/- GRK be part of the next PR?
esguerra>> The +GRK is what's on now. Will add the model and import data for next PR.

**Review by AJK-dev — APPROVED** [review]

**Pull request merged**